src/Store/SingleTableStore.php

```diff
@@ -29,6 +28,7 @@ public function __construct(Connection $connection, array $aggregates)
         parent::__construct($connection);
 
         $this->aggregates = $aggregates;
+        $this->tableName = 'eventstore';
```

Comment:
why didn't you add it via the constructor signature?

Reply:
the plan was to have the same signature like before & multi table store, but you are right with the comment below. The damage could be devastating if someone change the name of the table in a running system. So i guess its way better to provide the value via ctor but i would then drop the default name since defaulting parameters always tend to be a maintenance burden, wdyt?